### PR TITLE
adding a link

### PR DESCRIPTION
**Pull request by Deland-Han**
Adding a link to how to enable Microsoft Whiteboard on Surface Hub.

### CODE REVIEW

File: devices/surface-hub/index.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Surface Hub 2S is an all-in-one digital interactive whiteboard, meetings platfor
                         <p><a href="https://techcommunity.microsoft.com/t5/Surface-IT-Pro-Blog/Behind-the-design-Surface-Hub-2S/ba-p/464099" target="_blank">Behind the design: Surface Hub 2S</a></p>
                         <p><a href="surface-hub-2s-whats-new.md">What's new in Surface Hub 2S</a></p>
                         <p><a href="differences-between-surface-hub-and-windows-10-enterprise.md">Operating system essentials</a></p>
+                        <p><a href="https://support.office.com/en-us/article/enable-microsoft-whiteboard-on-surface-hub-b5df4539-f735-42ff-b22a-0f5e21be7627">Enable Microsoft Whiteboard on Surface Hub</a></p>
```
Comment (illfated) [inline devices/surface-hub/index.md]:
```suggestion
                        <p><a href="https://support.office.com/article/enable-microsoft-whiteboard-on-surface-hub-b5df4539-f735-42ff-b22a-0f5e21be7627">Enable Microsoft Whiteboard on Surface Hub</a></p>
```
This page is available for translation into multiple languages.
Language agnostic links are preferred, unless proven only to work when using `/en-us/`.

**Comment by JarrettRenshaw** [comment]
#sign-off

**Comment by illfated** [comment]
@Deland-Han : if you would be so kind, please merge the suggestion I made above. 👍

File: devices/surface-hub/index.md (hunk 1 above)
Comment (illfated) [reply devices/surface-hub/index.md]: OK, I see you added the change directly. Please mark this conversation as **resolved** to hide it.

**Comment by Deland-Han** [comment]
@illfated Sure. We just decided to use another link instead. I have updated the pull request,

**Comment by illfated** [comment]
Fair enough. that works too. I will remove my suggestion above to clean up this comment flow.

**Pull request merged**